reactioncommerce/reaction
# Inventory Tracking Backorders 1935

Resolves #1935 

**Description** 
Only added one of the purchased items when "Deny when out of stock" is disabled.

**Test**
1. Create product
2. Add a top level variant
3. Add 1 quantity
4. Disable Deny when out of stock
5. Publish product and variant
6. Purchase 2 or more of that variant
7. Check cart has the amount you have added.
